cloud-stream-custom-queue-names/src/main/resources/application.yaml

```diff
@@ -0,0 +1,48 @@
+spring:
+  cloud:
+    function:
+      definition: uppercase
+    stream:
+      bindings:
+        uppercase-in-0:
+          destination: manualackqueue
+          binder: solace-broker
+          group: myconsumergroup
+          consumer:
+            concurrency: 1
+        uppercase-out-0:
+          destination: uppercase/topic
+          binder: solace-broker
+      binders:
+        solace-broker:
+          type: solace
+          environment:
+            solace:
+              java:
+                host: 'tcp://localhost:55554'
+                msgVpn: default
+                clientUsername: default
+                clientPassword: default
+                connectRetries: -1
+                reconnectRetries: -1
+      solace:
+        bindings:
```

Comment:
Consider adding another binding with its own custom queue name that is just a literal to show how simple it is.

Reply:
Added